packages/int_datetime/lib/int_datetime.dart

```diff
@@ -15,14 +15,22 @@ int get dtNow => DateTime.now().millisecondsSinceEpoch;
 int get dtDay {
   final now = DateTime.now();
 
-  return DateTime.utc(now.year, now.month, now.day).millisecondsSinceEpoch;
+  return DateTime.utc(
+    now.year,
+    now.day,
+    now.month,
+  ).millisecondsSinceEpoch;
 }
 
 /// Will return time of the start of current month in milliseconds since epoch.
 int get dtMonth {
   final now = DateTime.now();
 
-  return DateTime.utc(now.year, now.month).millisecondsSinceEpoch;
+  return DateTime.utc(
```

Comment:
Same here. Please make sure the order is the right one

Reply:
It is also temporary code I saved =)